sourcecode/apis/contentauthor/app/Libraries/H5P/Adapters/CerpusH5PAdapter.php

```diff
@@ -21,7 +20,12 @@ class CerpusH5PAdapter implements H5PAdapterInterface
      */
     public function alterParameters($parameters, H5PAlterParametersSettingsDataObject $settings = null)
     {
-        return QuestionBankClient::convertMathToInlineDisplay($parameters);
+        $pattern = [
```

Comment:
Might be prudent to leave a comment indicating what this is supposed to do, since there's no longer a function name indicating the purpose.